docs(feat): smtp for elastic metal MTA-1952 (#590)
scaleway/docs-content: compute/elastic-metal/how-to/enable-smtp.mdx

---
meta:
  title: How to enable STMP for Elastic Metal servers
  description: This page explains how to enable STMP for Elastic Metal servers
content:
  h1: How to enable STMP for Elastic Metal servers
  paragraph: This page explains how to enable STMP for Elastic Metal servers
dates:
  validation: 2022-05-04
  posted: 2022-05-04
---

By default, the SMTP ports (25, 465, 587) of your Elastic Metal server are blocked, and cannot be reached from our infrastructure. 

> If you want to send e-mails from your server, you can enable SMTP for each Available Zone from the console.

Reviewer: add link to AZ concept?
```suggestion
If you want to send e-mails from your server, you can enable SMTP for each Availability Zone from the console.
```

> If you want to send e-mails from your server, you can enable SMTP for each Availability Zone from the console.

Reviewer 2: ```suggestion
If you want to send emails from your server, you can enable SMTP for each Availability Zone from the console.
```

<Message type="requirement">

- You have an account and are logged into the [Scaleway console](https://console.scaleway.com) 
- You have [created](/compute/elastic-metal/how-to/create-server) and [installed](/compute/elastic-metal/how-to/install-server) an Elastic Metal server
- You have [verified your identity](/console/my-account/how-to/verify-identity)

</Message>

<Message type="tip"> 

> This guide explains how to enable SMTP on Elastic Metal servers. If you want to enable SMTP on an Instance, follow our doucmentation [how to send emails from your Instance](compute/instances/how-to/send-emails-from-your-instance).

Reviewer: ```suggestion
This guide explains how to enable SMTP on Elastic Metal servers. If you want to enable SMTP on an Instance, follow our documentation on [how to send emails from your Instance](compute/instances/how-to/send-emails-from-your-instance).
```

</Message>

> 1. Click **Elastic Metal** in the Compute section of the side menu in the [Scaleway console](https://console.scaleway.com).

Reviewer: ```suggestion
1. Click **Elastic Metal** in the **Compute** section on the side menu of the [Scaleway console](https://console.scaleway.com).
```

2. Click the **General Settings** tab. The SMTP configuration displays.

    <Message type="important">

    You have to [verify your identity](/console/my-account/how-to/verify-identity) before you can enable the SMTP ports.

    </Message>

> 3. Toggle the switch to **on** for each Available Zone you want to enable STMP.

Reviewer: ```suggestion
3. Toggle the switch to **ON** beside each Availability Zone for which you want to enable STMP.
```

    <Lightbox src="scaleway-enable-smtp.webp" alt="" />

    <Message type="note">

>     If SMTP is not available for a specific Available Zone, the zone will display as greyed-out. 

Reviewer: ```suggestion
    If SMTP is not available for a specific Availability Zone, the zone will display as greyed-out. 
```
